src/group_theory/perm/cycle/basic.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2019 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes, Yaël Dillies
 -/
+import algebra.module.big_operators
```

Comment:
I'm a bit wary of this import, since this file is a transitive dependency of `group_theory.perm.fin` which feels rather basic to me, and `algebra.module.big_operators` itself has a lot of transitive dependencies. So I'd rather see this material in a new file.

Reply:
As I mentioned to you on Zulip, this file is in general too big, so I am planning on splitting it in an upcoming PR. But in the meantime I can't tell how to split up appropriately.